authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/filter/IdAuthFilter.java

```diff
@@ -993,16 +992,9 @@ protected boolean isTrustValidationRequired() {
 	 * @return
 	 */
 	public Set<String> getSystemSupportedLanguageCodes() {
-		Set<String> systemSupportedLanguges;
 		String languages = env.getProperty(IdAuthConfigKeyConstants.MOSIP_MANDATORY_LANGUAGES) + ","
-				+ env.getProperty(IdAuthConfigKeyConstants.MOSIP_OPTIONAL_LANGUAGES);
-		if (null != languages && languages.contains(",")) {
-			systemSupportedLanguges = Arrays.stream(languages.split(",")).collect(Collectors.toSet());
-		} else {
-			systemSupportedLanguges = new HashSet<>();
-			systemSupportedLanguges.add(languages);
-		}
-		return systemSupportedLanguges;
+				+ env.getProperty(IdAuthConfigKeyConstants.MOSIP_OPTIONAL_LANGUAGES);		
+		return new HashSet<>(List.of(languages.split(",")));
```

Comment:
Please change this return type to List instead of Set.

Reply:
Here can we use Set only. Here we are not required the order of elements.